src/web/handlers.rs

```diff
@@ -60,9 +60,16 @@ pub async fn get_tiles(
         // tags.clone().commit(&mut request.extensions_mut());
     }
 
+    // Only set the region if the country is US
+    // see "Instant Find | API Response Caching" in the 7_jul_21 spec.
```

Comment:
I think adM wanted us to keep the current use of "region-code" for non-US countries in the API request and the cache key. I am confirming this with them again now, let's see what they say.

Reply:
Confirmed that we should continue sending/caching "region-code" for all the countries. adM will update their API doc to reflect this.